### PR TITLE
i18n(ko-KR): update `tutorial/`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- update `tutorial/`
- #9240
- #10209 
- #10223

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by jsparkdev** [comment]
The changes in this PR are included in #10258. I will close this PR when #10258 is merged.

**Review by yanthomasdev — APPROVED** [review]
LGTM! I'll merge the umbrella PR once I approve all of the individual PRs.

**Pull request closed without merging**
